feature/experimental/isis/otg_tests/internal/session/session.go

```diff
@@ -255,7 +255,14 @@ func (s *TestSession) PushDUT(ctx context.Context) error {
 		return fmt.Errorf("configuring network instance: %w", err)
 	}
 	dutConf := s.DUTConf.GetOrCreateNetworkInstance(*deviations.DefaultNetworkInstance).GetOrCreateProtocol(PTISIS, ISISName)
-	_, err := ygnmi.Replace(ctx, s.DUTClient, ProtocolPath().Config(), dutConf)
+	// Clear ISIS Protocol
```

Comment:
same thing for OTG.

Reply:
We need to clear existing protocol and counters /stats will be resetted before the test starts.